samples/account-linking/csharp/.tours/tab-account-linking.tour

```diff
@@ -0,0 +1,221 @@
+{
+  "$schema": "https://aka.ms/codetour-schema",
+  "title": "Tab-Account-Linking",
+  "steps": [
+    {
+      "title": "Introduction",
+      "description": "# Introduction\r\nThis is a tour of the Tab's usage of the account linking for calling the [Github Repos API](https://docs.github.com/en/rest/reference/repos#list-repositories-for-a-user).\r\n\r\nAt a high level this uses the backend service as a proxy for the /repositories API."
+    },
+    {
+      "file": "Source/wwwroot/index.html",
+      "description": "# The Tab Page\r\nThis is an extremely minimal tab implementation. In practice you will likely build a more complex single-page application but to keep this focused on the core functionality this is built using pure js & the [Microsoft Teams client sdk](https://docs.microsoft.com/en-us/javascript/api/overview/msteams-client?view=msteams-client-js-latest).\r\n\r\nThis app just dumps the JSON payload of the Repositories API into the body of the tab\r\n\r\n![Screenshot of json payload of GitHub repositories displayed in a Teams Personal tab](./Docs/Images/TabScreenshot.jpg)",
+      "line": 11,
+      "selection": {
+        "start": {
+          "line": 6,
+          "character": 4
+        },
+        "end": {
+          "line": 11,
+          "character": 12
+        }
+      }
+    },
+    {
+      "file": "Source/wwwroot/scripts.js",
+      "description": "# JavaScript entrypoint\r\nThe entry point of the tab registers handlers for the login / logout buttons and initially calls the 'onLogin' method to automatically kick off the login / repository fetching when the user visits the page. ",
+      "line": 92
+    },
+    {
+      "file": "Source/wwwroot/scripts.js",
+      "description": "# On Log In\r\nThis is the primary functionality of the frontend. We will trace this call first as if the user _isn't_ logged in.",
+      "line": 49,
+      "selection": {
+        "start": {
+          "line": 79,
+          "character": 25
+        },
+        "end": {
+          "line": 79,
+          "character": 32
+        }
+      }
+    },
+    {
+      "file": "Source/wwwroot/scripts.js",
+      "description": "This is a wrapper for the `microsoftTeams.authentication.getAuthToken` call, this will return an Azure AD access token for the configured backend API. (Assuming the manifest & AzureAd application registrations are configured correctly)",
+      "line": 51,
+      "selection": {
+        "start": {
+          "line": 31,
+          "character": 3
+        },
+        "end": {
+          "line": 31,
+          "character": 46
+        }
+      }
+    },
+    {
+      "file": "Source/wwwroot/scripts.js",
+      "description": "We then call the `/github/repositories` API on our web API with the `Bearer` token attached as the authorization header per the [OAuth2.0 specification](https://datatracker.ietf.org/doc/html/rfc6750#section-2.1). ",
+      "line": 58,
```

Comment:
Similarly, you must have added something above and pushed everything down. Looks like there is some CodeTour build breakers that we could run to help prevent these from breaking (https://marketplace.visualstudio.com/items?itemName=vsls-contrib.codetour#maintaining-tours)

Reply:
Same for some scripts.js below too